meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mqevk/bootloader.uuu.in

```diff
@@ -0,0 +1,30 @@
+uuu_version 1.2.39
+
+SDP: boot -f imx-boot-mfgtool
+
+# This command will be run when ROM supports stream mode
+SDPS: boot -f imx-boot-mfgtool
+
+# These commands will be run when use SPL and will be skipped if no spl
+# SDPU will be deprecated. please use SDPV instead of SDPU
```

Comment:
Do we need the SDPU lines here as we're now using SDPV?

Reply:
I was keeping them as older uuu binaries might need them.  But since we're supplying the uuu binary version, I'll remove.